cli/src/cluster_query.rs

```diff
@@ -1780,8 +1776,39 @@ pub fn process_show_stakes(
 ) -> ProcessResult {
     use crate::stake::build_stake_state;
 
-    let progress_bar = new_spinner_progress_bar();
-    progress_bar.set_message("Fetching stake accounts...");
+    let vote_account_progress_bar = new_spinner_progress_bar();
+    vote_account_progress_bar.set_message("Searching for matching vote accounts...");
```

Comment:
Let's move this inside the `Some` case, as it will be confusing when a user hasn't provided any keys.

Reply:
Done